v0.5/fastn-lang/src/f-script/README.md

```diff
@@ -0,0 +1,221 @@
+# F-script appearances
+
+- body of a function definition
+
+```ftd
+-- string foo():
+string name:
+
+if name == "" {
+  "<default>"
+} else {
+  name
+}
+```
+
+The body contains a list of expressions. The last expression is evaluated and
+returned. There's no return keyword, if the function type is `void` then
```

Comment:
> There's no return keyword.

This is wrong. We need early return statement.

Reply:
We can note `void` is same as `()` in Rust.

Reply:
This is done in https://github.com/fastn-stack/fastn/pull/2010/commits/2ab92c8f526a35437e6b0e5e55709bc1fb1d54d1

Reply:
> We can note void is same as () in Rust.
I'll ignore this for now.